src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas2SubmittedApplicationReport.kt

```diff
@@ -22,8 +22,11 @@ interface Cas2SubmittedApplicationReportRepository : JpaRepository<DomainEventEn
         TO_CHAR(
           CAST(events.data -> 'eventDetails' ->> 'submittedAt' AS TIMESTAMP),
           'YYYY-MM-DD"T"HH24:MI:SS'
-        ) AS submittedAt
+        ) AS submittedAt,
+        TO_CHAR(applications.created_at, 'YYYY-MM-DD"T"HH24:MI:SS') AS createdAt
       FROM domain_events events
+      JOIN cas_2_applications applications
+      ON events.application_id = applications.id      
       WHERE events.type = 'CAS2_APPLICATION_SUBMITTED'
         AND events.occurred_at  > CURRENT_DATE - 365
```

Comment:
> A further option might be to include created_at within the event details so this data is also persisted to the domain_events table.

agreed, that does seem like it would be a good improvement (if only to save so much extra setup in the tests 🙈 ). 
I wonder if we might be running out of a time a bit so might be an improvement for @tahb to prioritise ... I guess you will have to be changing the [shape of the status update domain events for this ticket](https://dsdmoj.atlassian.net/jira/software/c/projects/CAS2/boards/1347/backlog?selectedIssue=CAS2-115), so could be done after that...

Reply:
Added new ticket to capture this requirement: https://dsdmoj.atlassian.net/browse/CAS2-260